app/controllers/checkouts_controller.rb

```diff
@@ -133,13 +133,8 @@ def time_remaining(checkout)
     end
 
     def checkout_actions(checkout)
-      if checkout.return_time > DateTime.current
-        view_context.link_to('Return', return_checkout_url(checkout), class: 'btn btn-outline btn-xs', method: :patch)
-      elsif checkout.return_time < DateTime.current && checkout.media_object.lending_status == 'available'
-        view_context.link_to('Checkout', checkouts_url(params: { checkout: { media_object_id: checkout.media_object_id } }), class: 'btn btn-primary btn-xs', method: :post)
-      else
-        ''
-      end
+      return unless checkout.return_time > DateTime.current
```

Comment:
Is `nil` an okay return value or should it be empty string like it was before?

Reply:
`nil` wasn't causing issue on my local instance but thinking on it, using an empty string is probably safer. Preemptively avoid some weird edge case/browser issue where it decides it doesn't like `nil`... Will change it to return empty string.